build/package/LICENSE

```diff
@@ -1788,3 +1788,370 @@ licenses.
     software is granted. This software is provided "as is" without
     express or implied warranty, and with no claim as to its
     suitability for any purpose.
+
```

Comment:
Is there any order for how we organize this file? It looks like all the node dependencies are listed first, followed by all the jar dependencies. Do we want to move these all up towards the top? Or is it not worth the effort?

Reply:
Not really I kind of was just adding to the end but I will rearrange that all node stuff listed before scala/jar